easy-ui-react/src/RadioButtonGroup/RadioButtonGroup.test.tsx

```diff
@@ -0,0 +1,113 @@
+import { screen } from "@testing-library/react";
+import React from "react";
+import { vi } from "vitest";
+import type { Key } from "react-aria-components";
+import { render, selectCheckbox } from "../utilities/test";
+import { RadioButtonGroup } from "./RadioButtonGroup";
+
+describe("<RadioButtonGroup />", () => {
+  beforeEach(() => {
+    vi.useFakeTimers();
+  });
+
+  afterEach(() => {
+    vi.useRealTimers();
+  });
+  it("should render a radio group", () => {
+    render(
+      <RadioButtonGroup>
+        <RadioButtonGroup.Button id="first">
+          First Button
+        </RadioButtonGroup.Button>
+        <RadioButtonGroup.Button id="second">
+          Second Button
+        </RadioButtonGroup.Button>
+      </RadioButtonGroup>,
+    );
+    expect(screen.getByRole("radiogroup")).toBeInTheDocument();
+    expect(screen.getAllByRole("radio").length).toBe(2);
+    expect(screen.getAllByRole("radio")[0]).toHaveAttribute("aria-checked");
+  });
+
+  it("should update when a button is clicked", async () => {
+    const { user } = render(
+      <RadioButtonGroup defaultSelectedKeys={["first"]}>
+        <RadioButtonGroup.Button id="first">First item</RadioButtonGroup.Button>
+        <RadioButtonGroup.Button id="second">
+          Second item
+        </RadioButtonGroup.Button>
+      </RadioButtonGroup>,
+    );
+    const radios = screen.getAllByRole("radio");
+    expect(radios[0]).toBeChecked();
+
+    await selectCheckbox(user, radios[1]);
+    expect(radios[0]).not.toBeChecked();
+    expect(radios[1]).toBeChecked();
+  });
+
+  it("should support disabling the group", async () => {
+    const { user } = render(
+      <RadioButtonGroup defaultSelectedKeys={["first"]} isDisabled>
+        <RadioButtonGroup.Button id="first">First item</RadioButtonGroup.Button>
+        <RadioButtonGroup.Button id="second">
+          Second item
+        </RadioButtonGroup.Button>
+      </RadioButtonGroup>,
+    );
+    const radios = screen.getAllByRole("radio");
+    expect(radios[0]).toBeChecked();
+
+    await selectCheckbox(user, radios[1]);
+    expect(radios[0]).toBeChecked();
+    expect(radios[1]).not.toBeChecked();
+  });
+
+  it("should support disabling individual buttons", async () => {
+    const { user } = render(
+      <RadioButtonGroup defaultSelectedKeys={["first"]} isDisabled>
+        <RadioButtonGroup.Button id="first">First item</RadioButtonGroup.Button>
+        <RadioButtonGroup.Button id="second" isDisabled>
+          Second item
+        </RadioButtonGroup.Button>
+      </RadioButtonGroup>,
+    );
+    const radios = screen.getAllByRole("radio");
+    expect(radios[0]).toBeChecked();
+
+    await selectCheckbox(user, radios[1]);
+    expect(radios[0]).toBeChecked();
+    expect(radios[1]).not.toBeChecked();
+  });
+
+  it("should support controlled use", async () => {
+    const handleChange = vi.fn();
+    const { user, rerender } = render(
+      <RadioButtonGroup
+        selectedKeys={new Set<Key>(["second"])}
```

Comment:
can we keep array and set consistent between `defaultSelectedKeys` and `selectedKeys`? i think passing in an array for both properties would be good instead of a `Set`